packages/relay-sdk/src/index.ts

```diff
@@ -0,0 +1,132 @@
+     * 
+     * ```typescript
+     * const client = await LitRelayClient.connect('habanero', 'my-api-key', 'my-payer-secret');
+     * const result = await client.addPayee('payee-wallet-address');
+     * ```
+     * @param payeeAddress
+     * 
+     * @returns Promise<{ tokenId: string } | Error>
+     */
+    public addPayee(payeeAddress: string): Promise<{ tokenId: string }> {
+        if (!this.payerSecret) {
+            return Promise.reject('Payer secret key is missing');
+        }
+
+        return fetch(`${this.baseUrl}/add-users`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                'api-key': this.apiKey,
+                'payer-secret-key': this.payerSecret,
+            },
+            body: JSON.stringify([payeeAddress]),
+        })
+            .then(assertStatus(200, 'Failed to add payee: request failed'))
+            .then(res => res.json())
+            .then(json => assertString(json.tokenId, 'Failed to add payee: missing token ID'))
+            .then(tokenId => ({ tokenId }));
+    }
+
+    /**
+     * Registers a new payer with the Lit Relay server using the provided API key. Returns
+     * a new LitRelayClient instance with the payer secret key.
+     * 
+     * ```typescript
+     * const client = await LitRelayClient.register('habanero', 'my-api-key');
+     * ```
+     * 
+     * @param network 
+     * @param apiKey 
+     * 
+     * @returns LitRelayClient
+     */
+    public static register(network: SupportedNetworks, apiKey: string): Promise<LitRelayClient {
+        if (network !== 'habanero' && network !== 'manzano') {
+            throw Promise.reject('Invalid network');
+        }
+
+        const baseUrl = getRelayURLByNetwork(network);
+
+        return fetch(`${baseUrl}/register-payer`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                'api-key': apiKey,
+            },
+        })
+            .then(assertStatus(200, 'Failed to register payer: request failed'))
+            .then(res => res.json())
+            .then(json => getProp(json, 'payerSecretKey'))
+            .then(json => assertString(json.payerSecretKey, 'Failed to register payer: missing secret key'))
+            .then(payerSecretKey => new LitRelayClient(baseUrl, apiKey, payerSecretKey));
+    }
+
+    /**
+     * Connects to the Relay server for the specified network using the provided API key and payer secret
+     * and returns a new LitRelayClient instance.
+     * 
+     * ```typescript
+     * const client = await LitRelayClient.connect('habanero', 'my-api-key', 'my-payer-secret');
+     * ```
+     * 
+     * @param network 
+     * @param apiKey 
+     * @param payerSecret 
+     * 
+     * @returns LitRelayClient
+     */
+    public static connect(network: SupportedNetworks, apiKey: string, payerSecret: string) {
+        if (network !== 'habanero' && network !== 'manzano') {
```

Comment:
Since we run this check in all methods, it'd be nice if it was standardized to a single assert function